components/Themes/src/MDCColorScheme.h

```diff
@@ -14,42 +14,92 @@
  limitations under the License.
  */
 
-#import <UIKit/UIKit.h>
-
 /**
  A color scheme comprised of set of primary and secondary colors. Material design guidelines
  recommend using primary and secondary colors with light and dark color variants.
  */
-@interface MDCColorScheme : NSObject
+@protocol MDCColorScheme
```

Comment:
I _think_ we're using gerund naming for protocols, so `MDCColorScheming`? :) @ianegordon ?